lib/services/common.dart

```diff
@@ -4,9 +4,9 @@
 
 library dart_pad.common;
 
-final String serverURL = 'https://dart-services.appspot.com/';
-//final String serverURL = 'http://doc-test.dart-services.appspot.com/';
-//final String serverURL = 'http://127.0.0.1:8082/';
+//final String serverURL = 'https://dart-services.appspot.com/';
+final String serverURL = 'http://doc-test.dart-services.appspot.com/';
```

Comment:
Maybe leave it in, so that you guys test the PR, and then I rebase the PR when this is landed at dart-services ?